ci/build.sh

```diff
@@ -70,7 +70,7 @@ echo "Test docker image locally..."
 ruby deploy/test/test_docker.rb
 
 # Check for changes that require re-generating overrides yaml files
-if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_EVENT_TYPE" == "push" ] && [ -n "$GITHUB_TOKEN" ]; then
+if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
```

Comment:
Ah, @samjsong , PRs submitted from outside repos like https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/264 wont' have `$GITHUB_TOKEN` .  So this will require some more thought.